docs/metrics/getting-started-gauge/Program.cs

```diff
@@ -23,27 +23,21 @@
 
 public class Program
 {
-    private static readonly Meter MyMeter = new Meter("TestMeter", "0.0.1");
+    private static readonly Meter MyMeter = new Meter("MyMeter", "0.0.1");
     private static readonly Random RandomGenerator = new Random();
 
     public static async Task Main(string[] args)
     {
         using var meterProvider = Sdk.CreateMeterProviderBuilder()
-                .AddSource("TestMeter")
+                .AddSource("MyMeter")
                 .AddConsoleExporter()
                 .Build();
 
         ObservableGauge<long> gauge = MyMeter.CreateObservableGauge<long>(
-            "Gauge",
-            () =>
+            "MyGauge",
+            () => new List<Measurement<long>>()
             {
-                var tag1 = new KeyValuePair<string, object>("tag1", "value1");
-                var tag2 = new KeyValuePair<string, object>("tag2", "value2");
-
-                return new List<Measurement<long>>()
-                {
-                    new Measurement<long>(RandomGenerator.Next(1, 1000), tag1, tag2),
-                };
+                new (RandomGenerator.Next(1, 1000), new ("tag1", "value1"), new ("tag2", "value2")),
```

Comment:
we may use `Process.GetCurrentProcess().WorkingSet64`

Reply:
Let me update that.

Reply:
Just tried and it's pretty boring, the numbers remain the same (I guess it's partially due to the fact that CLR pre allocates).

Reply:
Our sdk is good at *not* leaking memory ;)